kvirt/vbox/__init__.py

```diff
@@ -928,7 +928,7 @@ def add_image(self, image, pool, cmd=None):
         os.system(cmd)
         return {'result': 'success'}
 
-    def create_network(self, name, cidr, dhcp=True, nat=True):
+    def create_network(self, name, cidr, dhcp=True, nat=True, domain=''):
```

Comment:
good point to include it for the vbox provider.
you also want to edit the base.py class which indicates all methods ( and their signatures) a provider should implement